_data/retail.yml

```
@@ -220,25 +259,26 @@ websites:
     url: https://www.jbhifi.com.au/
     twitter: JBHiFi
     img: jbhifi.png
-
-  - name: Jet
-    url: https://jet.com
-    twitter: JetHeads
-    img: jet.png
+    regions:
+      - au
 
   - name: jpc
     url: https://www.jpc.de/
     twitter: jpc_de
     facebook: jpc.de
     img: jpc.png
     lang: de
+    regions:
+      - de
```

```suggestion
    lang: de
```
jpc [ships internationally](https://www.jpc.de/jpcng/home/static/-/page/porto.html).